### ISSUE TITLE
Recovery Strategy IDs

### ISSUE THREAD

**Issue by tscottjendev** [Jersey-Electricity-plc/jwissues#40]
Has this concept been replaced by Reminders?

**Comment by RPCJendev** [comment]
Yes that is correct. It would be worth running through the recovery strategy on the NAV2009R2 versions with you to see the concept that was originally done.
